HW.txt

```diff
@@ -0,0 +1,27 @@
+function cal(num1,num2)  {
+ 
+var num1 = 1
+var num2 = 2 
+
+if (num1 + num2) {
+return "added"
+}
+
+if (num1 - num2){
+return "substract"
+}
+
+if (num1 / num2 ){
+return "divide"
+}
+
+if (num1 *num2){
+return "multiply"
+}
+else if ("caterpillar") {
+return "rallipretac"
+}
+
+return num1 
+return num2
+}
```

Comment:
1. For the calculator, nice try but what we want to accomplish it to receive 3 argumets `num1, num2, and the operation`. So our function should look something similar to
```js
function opt(num1,num2,op){

	if (op === "add") {
	return num1+num2;
        }

	if (op === "subtract"){
		return num1 - num2;
        }

	if (op === "multiply"){
		return num1 * num2;
        }

	if (op === "divide"){
		return num1 / num2;
        }
}
```